recipes/dogpile.cache/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "dogpile.cache" %}
+{% set version = "0.6.4" %}
+{% set md5 = "66e0a6cae6c08cb1ea25f89d0eadfeb0" %}
```

Comment:
We prefer the sha256 hash which can be found here, https://pypi.org/project/dogpile.cache/#files